hathor/transaction/storage/transaction_storage.py

```diff
@@ -563,10 +563,16 @@ def get_metadata(self, hash_bytes: bytes) -> Optional[TransactionMetadata]:
         except TransactionDoesNotExist:
             return None
 
+    def get_all_transactions(self) -> Iterator[BaseTransaction]:
+        """Return all transactions/blocks.
```

Comment:
Suggested docstring: "Return all vertices (transactions and blocks) within the allowed scope."

Reply:
Thanks!